numpy/random/tests/test_extending.py

```diff
@@ -42,10 +43,37 @@
 @pytest.mark.skipif(cython is None, reason="requires cython")
 @pytest.mark.slow
```

Comment:
Pretty sure it will leak references (since it cannot unload the so), could add `@pytest.mark.leaks_references` (although I have to fix that decorator up, since pytest-leaks now has a preferred name). But honestly, do not worry about it, I can add it whenever I notice it next time, and I rarely run the slow tests through the refcount checker...